allennlp/predictors/semantic_role_labeler.py

```diff
@@ -198,16 +231,17 @@ def predict_json(self, inputs: JsonDict) -> JsonDict:
         if not instances:
             return sanitize({"verbs": [], "words": self._tokenizer.split_words(inputs["sentence"])})
 
-        outputs = self._model.forward_on_instances(instances)
+        return self.predict_instances(instances)
 
-        results = {"verbs": [], "words": outputs[0]["words"]}
-        for output in outputs:
-            tags = output['tags']
-            description = self.make_srl_string(output["words"], tags)
-            results["verbs"].append({
-                    "verb": output["verb"],
-                    "description": description,
-                    "tags": tags,
-            })
+    def predict_words_list(self, words_list: List[str]) -> JsonDict:
```

Comment:
Again the logic in this method should just be moved into `predict_tokenized` - this is a duplicate method.